self_hosting_machinery/webgui/selfhost_model_assigner.py

```diff
@@ -139,10 +139,15 @@ def first_run(self):
             "model_assign": {
                 "Refact/1.6B": {
                     'gpus_shard': 1,
-                    'share_gpu': False,
-                }
+                    'share_gpu': True,
+                },
+                "thenlper/gte-base": {
+                    'gpus_shard': 1,
+                    'share_gpu': True,
+                },
             },
             "completion": "Refact/1.6B",
+            "embeddings": "thenlper/gte-base",
```

Comment:
why do we need embeddings field here? looks like default_embeddings_model in caps is enought